data/json/monster_special_attacks/spells.json

```
@@ -674,6 +674,21 @@
     "id": "portal_cage_revert_transform",
     "furniture": [ { "result": [ "f_null" ], "valid_furniture": [ "f_absence" ] } ]
   },
+  {
+    "id": "skunk_squirt",
+    "type": "SPELL",
+    "name": { "str": "Skunk Spray" },
+    "description": "Skunk spray defense mechanism.",
+    "valid_targets": [ "hostile" ],
+    "effect": "attack",
+    "shape": "cone",
+    "min_range": 4,
+    "max_range": 4,
+    "//": "Duration is 3-5 hours, can't seem to write it in hours, had to write it in moves which is terrible",
+    "min_duration": 1080000,
+    "max_duration": 1800000,
```

```suggestion
    "max_duration": 1800000,
    "flags": [ "RANDOM_DURATION" ]
```
Since you want the duration be random, you should use the flag for this